### DISCUSSION TITLE
Options for start time and stop time per song?

### DISCUSSION THREAD

**Discussion by brandonvu99** [navidrome/navidrome#2336] [Ideas]
iTunes has options for each song to start and/or stop at a specific, user-defined timestamp. 
I was wondering if this would be feasible to add to navidrome?

Example: 
![itunes_start_stop_times](https://user-images.githubusercontent.com/24399083/235405917-2b3b612f-9643-4a10-96fc-9f091d2afe50.png)

**Comment by deluan** [comment]
Yes, I like this. May introduce this in the future, maybe after looking into the CUE support PR (#2201)

**Comment by mgstreak** [comment]
I’m wondering if this idea has been (or could be) implemented?
